scene/gui/graph_edit.cpp

```diff
@@ -351,7 +351,7 @@ void GraphEdit::_graph_node_raised(Node *p_gn) {
 	if (gn->is_comment()) {
 		move_child(gn, 0);
 	} else {
-		gn->raise();
+		gn->move_to_top();
```

Comment:
Aren't the two branches equivalent here?

Reply:
Actually no, `move_to_top` seems to move to the *bottom* of the branch. That makes the new name pretty confusing IMO.

Reply:
`move_to_top()` moves node on top of others visually, hence the name. Maybe this method shouldn't exist outside CanvasItem 🤔

Reply:
move_to_visual_top or move_to_top_visual? I know it's getting wordy, but that way people won't get confused...

Reply:
I don't know if mentioning visuals in a Node method is a good idea.

Reply:
`front_to_show`
Instead of taking care of both side, it is better to focus on one.

Reply:
I don't think "top" makes any sense for the `Node` type, since it doesn't have any rendering at all.

I personally think the method should be removed, moved to the parent, or the behavior should exactly match the method name, best I could think of is `move_to_last_sibling`.